test/MSVC/PCH-source.py

```diff
@@ -40,7 +40,7 @@
 env = Environment(tools=['msvc', 'mslink'])
 env['PCH'] = env.PCH('Source1.cpp')[0]
 env['PCHSTOP'] = 'Header1.hpp'
-env.Program('foo', ['foo.cpp', 'Source2.cpp', 'Source1.cpp'])
+env.Program('foo', ['Source2.cpp', 'Source1.cpp', 'foo.cpp'])
```

Comment:
Maybe add a comment here that the order is chosen for a specific reason, so somebody doesn't resort it later.